packages/remix/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@sentry/remix",
+  "version": "7.1.1",
+  "description": "Official Sentry SDK for Remix",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/remix",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=14"
+  },
+  "private": true,
+  "dependencies": {
+    "@sentry/core": "7.0.0-beta.0",
+    "@sentry/hub": "7.0.0-beta.0",
+    "@sentry/integrations": "7.0.0-beta.0",
+    "@sentry/node": "7.0.0-beta.0",
+    "@sentry/react": "7.0.0-beta.0",
+    "@sentry/tracing": "7.0.0-beta.0",
+    "@sentry/utils": "7.0.0-beta.0",
```

Comment:
We'll need to update the versions here as well!

Reply:
Oops! Updated 👍